modules/csharp/pages/releasenotes.adoc

```diff
@@ -9,8 +9,12 @@ ifdef::prerelease[:page-status: {prerelease}]
 include::partial$_set_page_context_for_csharp.adoc[]
 
 [#maint-latest]
+include::partial$release-notes/couchbase-mobile-csharp-release-note.3.1.7.adoc[]
+
```

Comment:
Adding all this spacing is fine as long as it makes sense to you!
But makes me realise I didn't explain the original layout (e.g. that made sense to me) which was:

```
[#maint-latest]
include::...(latest include)

include::...(earlier ones, all bunched together)
include::...
include::...
```

e.g. just a single line that separates the current from the rest.

(Not at all a problem if you like the extra spaces though, just surprised me as I wasn't expecting it, and then I had to remember what I'd done before 🤷)

Reply:
Yeah I remembered the most recent needs to be tied to the label, just scans a bit easier for me with the spacing and how I've been drilled in the past